rasa/cli/x.py

```diff
@@ -140,8 +163,7 @@ def start_core_for_local_platform(args: argparse.Namespace, rasa_x_token: Text):
     vars(args).update(
         dict(
             nlu_model=None,
-            channel="rasa",
-            credentials="credentials.yml",
+            credentials=DEFAULT_CREDENTIALS_PATH,
```

Comment:
this shouldn't necessarily take the default should it?
```suggestion
            credentials=credentials_path,
```

Reply:
I think we can just drop it

Reply:
yes